<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSpinBox" name="numOfRows">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>281</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="getMinButton_2">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getAvrButton_2">
       <property name="text">
        <string>Avr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getMaxButton_2">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="mainTable">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>521</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="randomFill">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rand</string>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_numOfRows">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- кол. строк</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeOfSorting">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Пузырьком</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Быстрая</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Расчёска</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Гномья</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Обезьянья</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="textOut">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>180</y>
      <width>281</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="numOfColumns">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_numOfColumns">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;- кол. столбцов</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
